src/main/java/org/jenkinsci/plugins/ghprb/GhprbRootAction.java

```diff
@@ -50,11 +58,21 @@ public String getDisplayName() {
     public String getUrlName() {
         return URL;
     }
+    
+    public int getThreadCount() {
+        return triggerThreads == null ? 0 : triggerThreads.size();
+    }
+    
+    public GhprbRootAction() {
+        triggerThreads = Collections.newSetFromMap(new WeakHashMap<StartTrigger, Boolean>());
+        this.pool = Executors.newFixedThreadPool(10);
```

Comment:
You might want to make this configurable from a System property.  .NET CI will probably need a higher count.